src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -4386,6 +4386,41 @@ class spell_gen_azgalor_rain_of_fire_hellfire_citadel : public SpellScriptLoader
         }
 };
 
+// 26662 - Berserk
+class spell_gen_berserk : public SpellScriptLoader
+{
+    public:
+        spell_gen_berserk() : SpellScriptLoader("spell_gen_berserk") { }
+
+        class spell_gen_berserk_AuraScript : public AuraScript
+        {
+            PrepareAuraScript(spell_gen_berserk_AuraScript);
+
+            void OnApply(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+            {
+                GetTarget()->ApplySpellImmune(0, IMMUNITY_STATE, SPELL_AURA_MOD_TAUNT, true);
+                GetTarget()->ApplySpellImmune(0, IMMUNITY_EFFECT, SPELL_EFFECT_ATTACK_ME, true);
+            }
+
+            void OnRemove(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+            {
+                GetTarget()->ApplySpellImmune(0, IMMUNITY_STATE, SPELL_AURA_MOD_TAUNT, false);
+                GetTarget()->ApplySpellImmune(0, IMMUNITY_EFFECT, SPELL_EFFECT_ATTACK_ME, false);
```

Comment:
What if the creature has another aura which grants taunt immunity? Might these lines cause some issues?